10-setup-htcondor.sh

```diff
@@ -133,3 +217,54 @@ echo
 echo "Will use the following token(s):"
 condor_token_list
 echo
+
+cd $LOCAL_DIR
+
+# gwms files in the correct location
+cp -a /gwms/* .
+mkdir -p .gwms.d/bin 
+for target in cleanup  postjob  prejob  setup  setup_singularity; do
+    mkdir -p .gwms.d/exec/$target
+done
+cp -a /usr/sbin/osgvo-singularity-wrapper condor_job_wrapper.sh
+
+# minimum env to get glideinwms scripts to work
+export glidein_config=$LOCAL_DIR/glidein_config
+export condor_vars_file=$LOCAL_DIR/main/condor_vars.lst
+
+# set some defaults for the glideinwms based scripts
+if [[ -z $OSG_DEFAULT_CONTAINER_DISTRIBUTION ]]; then
+    OSG_DEFAULT_CONTAINER_DISTRIBUTION="70%__opensciencegrid/osgvo-el7:latest 30%__opensciencegrid/osgvo-el8:latest"
+fi
+cat >$glidein_config <<EOF
+ADD_CONFIG_LINE_SOURCE $PWD/add_config_line.source
+ALLOW_NONCVMFS_IMAGES True
```

Comment:
This one **must** be configurable, and off by default, at least until we document it. As we're seeing, non-CVMFS images use a lot of disk and network, and the site admin has no way of restricting this.

Reply:
Yeah, that is probably a mistake - let me take another test pass

Reply:
Fixed. I also added the latest stashcp, as that is an important component if /cvmfs it not available.